htdocs/cron/class/cronjob.class.php

```diff
@@ -1021,7 +1021,7 @@ function run_jobs($userlogin)
 				$result = call_user_func_array($this->methodename, $params_arr);
 			}
 
-			if ($result === false || $result != 0)
+			if ($result === false || $result < 0)
```

Comment:
@eldy 
yes but if my $result = "true", with this test, $result not equal "false" OR not equal 0 !
maybe the best test is : 

if ($result === false && $result != 0)

you see ?